integration_tests/e2e/add-steps.cy.ts

```diff
@@ -0,0 +1,174 @@
+import AddSteps from '../pages/add-steps'
+import { PlanType } from '../../server/@types/PlanType'
+import DataGenerator from '../support/DataGenerator'
+
+describe('Add Steps', () => {
+  const addStep = new AddSteps()
+
+  const selectStepDescriptionByIndex = (index: number) => {
+    return cy.get(`table.goal-summary-card__steps .govuk-table__body > :nth-child(${index}) > :nth-child(1)`)
+  }
+
+  const selectStepActorByIndex = (index: number) => {
+    return cy.get(`table.goal-summary-card__steps .govuk-table__body > :nth-child(${index}) > :nth-child(2)`)
+  }
```

Comment:
Is it possible to move these out of the `describe('Add steps,()...)` function? Still in this file, just not defining the function within the test itself.

Reply:
Addressed in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/124/commits/3e866574dd4374e959a1bf04f0401f6c8616474a